src/Components/ClusterRules/ClusterRules.cy.js

```diff
@@ -211,9 +214,24 @@ describe('cluster rules table', () => {
     ).forEach(([category, label]) => {
       SORTING_ORDERS.forEach((order) => {
         it(`${order} by ${label}`, () => {
+          let sortingParameter = category;
+          // modify sortingParameters for certain values
+          if (category === 'description') {
```

Comment:
If tests where passing previously, and now we are only modifying `impacted`, why do we need to modify the sorting parameter for all the fields and not only for `impacted`?

Reply:
You are correct, removed the "fluff" and left the "if" for the impacted category only